lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -307,6 +345,84 @@ public void OnCompleted()
         {
         }
 
+        private IConfiguration MapTaskConfiguration(IActiveContext activeContext, string taskId)
+        {
+            IConfiguration mapSpecificConfig;
+
+            if (!_perMapperConfiguration.TryPop(out mapSpecificConfig))
+            {
+                Exceptions.Throw(
+                    new Exception(string.Format("No per map configuration exist for the active context {0}",
+                        activeContext.Id)),
+                    Logger);
+            }
+
+            return TangFactory.GetTang()
+                .NewConfigurationBuilder(new[]
+                {
+                    TaskConfiguration.ConfigurationModule
+                        .Set(TaskConfiguration.Identifier, taskId)
+                        .Set(TaskConfiguration.Task, GenericType<MapTaskHost<TMapInput, TMapOutput>>.Class)
+                        .Build(),
+                    _configurationManager.MapFunctionConfiguration,
+                    mapSpecificConfig
+                })
+                .BindNamedParameter(typeof(InvokeGC), _invokeGC.ToString())
+                .Build();
+        }
+
+        private IConfiguration UpdateTaskConfiguration()
```

Comment:
Please add `Get` as a prefix to these methods. Or `Make`. The way they are stated, it sounds like they update the task configuration.